packages/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -192,6 +193,7 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
     this._tooltipId = getId('LineChartTooltipId_');
     this._rectId = getId('containerRectLD');
     this._staticHighlightCircle = getId('staticHighlightCircle');
+    this._createLegendsMemoized = memoizeFunction((data: LineChartDataWithIndex[]) => this._createLegends(data));
```

Comment:
why do we need to memoize this function. Is it called multiple times

Reply:
It is not called multiple times but it will help speed up the rendering during the update lifecycle of line chart triggered by user interaction of hover ,click etc.

Reply:
Can you compute any data on what gain this is bringing. Is it making the rendering or the computation more faster.